src/main/java/io/kestra/plugin/gcp/gcs/Trigger.java

```diff
@@ -93,7 +98,9 @@
         )
     }
 )
-public class Trigger extends AbstractTrigger implements PollingTriggerInterface, TriggerOutput<Downloads.Output>, GcpInterface, ListInterface, ActionInterface{
+public class Trigger extends AbstractTrigger implements PollingTriggerInterface, TriggerOutput<Downloads.Output>, GcpInterface, ListInterface, ActionInterface {
```

Comment:
You didn't implement this proposed changed by Anna in the issue to trace the change:

> Add `changeType` (`CREATE` or `UPDATE`) to trigger outputs (non-breaking).